security/tinc/src/opnsense/scripts/OPNsense/Tinc/lib/objects.py

```diff
@@ -106,6 +106,7 @@ def config_text(self):
                 result.append('ConnectTo = %s' % (host.get_hostname(),))
         result.append('Device=/dev/tinc%(id)s' % self._payload)
         result.append('Name=%(hostname)s' % self._payload)
+        result.append('Subnet=%s/32'%self.get_local_address().split('/')[0])
```

Comment:
@QDaniel I'm unsure about this, as it ends up in `tinc.conf` and I can't find a reason why you should want that, can you explain why we should do this?